### ISSUE TITLE
Why not use NormedLinear_Classifier?

### ISSUE THREAD

**Issue by GeoffreyChen777** [dvlab-research/Parametric-Contrastive-Learning#4]
Hi, thanks for your work!

I noticed that you didn't use NormedLinear_Classifier. In your model you just use a nn.Linear.

What's the reason?

Thanks!

**Comment by jiequancui** [comment]
Hi, Thank you for your question!
There are no special reasons. 
We observed that after the training, the weight of the classifier has been almost normalized.
